sycl/test-e2e/Matrix/joint_matrix_colA_rowB_colC_impl.hpp

```diff
@@ -0,0 +1,130 @@
+#include <iostream>
+#include <random>
+#include <sycl/sycl.hpp>
+
+using namespace sycl;
+using namespace sycl::ext::oneapi::experimental::matrix;
+using bfloat16 = sycl::ext::oneapi::bfloat16;
+
+#define TM 8
+#define TN SG_SZ
```

Comment:
Please, define TN in .cpp and decouple from SG_SZ, as they can be different in general case